v1.1/admin_ui_replication_dashboard.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Replication Dashboard
+toc: false
+feedback: false
+---
+
+CockroachDB Admin UI enables you to monitor the replication metrics for your cluster.
+
+<div id="toc"></div>
+
+To view the Replication metrics for your cluster, [access the Admin UI](explore-the-admin-ui.html#access-the-admin-ui), then from the Dashboard drop-down box, select **Replication**. 
+
+#### Cluster and Node View for the Time Series Graphs
```

Comment:
You go over this in the "Navigate the Admin UI" doc, right? I wouldn't repeat this guidance here or on other pages. It's a UI-wide behavior, so fine to just document it once.